c/extras/aggregator.cc

```diff
@@ -121,7 +121,16 @@ void Aggregator::aggregate_exemplars(DataTable* dt_exemplars,
   RowIndex ri_members = dt_members->sortby(cols, &gb_members);
   const int32_t* offsets = gb_members.offsets_r();
   arr32_t exemplar_indices(gb_members.ngroups());
-  const arr32_t ri_members_indices = ri_members.extract_as_array32();
+
+  const int32_t* ri_members_indices = nullptr;
+  arr32_t temp;
+
+  if (ri_members.isarr32()) {
+    ri_members_indices = ri_members.indices32();
+  } else {
+    temp = ri_members.extract_as_array32();
+    ri_members_indices = temp.data();
+  }
```

Comment:
@st-pasha is there any better way to convert `RowIndex` to `int32_t*` in a general case?